examples/lb3-application/src/migrate.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-lb3-application
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CoffeeShopApplication} from './application';
+
+export async function migrate(args: string[]) {
+  const existingSchema = args.includes('--rebuild') ? 'drop' : 'alter';
+  console.log('Migrating schemas (%s existing schema)', existingSchema);
+
+  const app = new CoffeeShopApplication();
+  await app.boot();
+  await app.migrateSchema({existingSchema});
```

Comment:
IIUC, this is not going to migrate LB3 models like `CoffeeShop`, `User`, `AccessToken`, etc. Auto-migration of LB3 models is out of scope of this pull request, but I feel it's important to clearly spell out the limitation of the current implementation. 

I am proposing to open a follow-up issue to keep track of this enhancement and modify `migrate.ts` to print a `console.warn` message, e.g.

```ts
console.warn('Skipping migration of models from LB3 application (not supported yet).');
console.warn('See https://github.com/strongloop/loopback/issues/<fill the number>');
await app.migrateSchema({existingSchema});

Reply:
Issue created: https://github.com/strongloop/loopback-next/issues/2825